src/In.ProjectEKA.DefaultHip/Resources/dignosticReportWithMediaDicomFileAsUrl.json

```diff
@@ -0,0 +1,55 @@
+{
+  "resourceType": "Bundle",
+  "id": "11123232324",
+  "type": "collection",
+  "entry": [{
+    "fullUrl": "123efdfzxa87",
+    "resource": {
+      "resourceType": "DiagnosticReport",
+      "id": "ultrasound",
+      "text": {
+        "status": "additional",
+        "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Unstructured data can be sent here</div>"
+      },
+      "status": "final",
+      "code": {
+        "text": "Abdominal Ultrasound"
+      },
+      "subject": {
+        "reference": "Patient/example"
+      },
+      "effectiveDateTime": "2012-12-01T12:00:00+01:00",
+      "issued": "2012-12-01T12:00:00+01:00",
+      "performer": [{
```

Comment:
if you are passing a reference, ideally such references should be included as another entry in the bundle. 
If not, then please add a "display" field. like 
{
   reference: "Practitioner/example",
   display: "Example Practitioner"
}